src/wp-includes/script-loader.php

```diff
@@ -2998,6 +2999,7 @@ function wp_enqueue_global_styles_css_custom_properties() {
  * @param int    $priority To set the priority for the add_action.
  */
 function wp_enqueue_block_support_styles( $style, $priority = 10 ) {
+	_deprecated_function( __FUNCTION__, '6.2.0', 'wp_style_engine_get_stylesheet_from_css_rules()' );
 	$action_hook_name = 'wp_footer';
```

Comment:
```suggestion

	$action_hook_name = 'wp_footer';
```

Empty new line to separate the deprecation from the original function's code.